components/Header/Header.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <StyledHeader>
+      <StyledH1>Movie-Box</StyledH1>
+    </StyledHeader>
+  );
+}
+
+const StyledHeader = styled.header`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 0 1rem;
+  height: 5rem;
+  position: sticky;
+  z-index: 100;
+`;
+
+const StyledH1 = styled.h1`
+  border: 2px solid var(--text-primary);
+  padding: 0.5rem 5rem;
+`;
```

Comment:
I noticed that you used 'rem' here for height and padding. 
If users prefer bigger texts, because they can't see well, they will increase the root-font-size (rem) in the settings of the browser. You should consider if you want the heights/paddings also changed if they do so.
I didn't test it, but maybe something like line 23 (padding 5rem) could destroy your style quite a bit.

Reply:
This really depends on the specific use-case, there is a very nice article about this topic 🤓:
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/